src/view/viewSpinner.js

```diff
@@ -0,0 +1,23 @@
+let rotateCount = 0;
+let startTime = null;
+
+export function spinSpinner(timestamp) {
+  const spinners = document.querySelectorAll('.spinner img');
+  if (!startTime) {
+    startTime = timestamp;
+  }
+  rotateCount = (timestamp - startTime) / 3;
+  if (rotateCount > 359) {
+    rotateCount %= 360;
+  }
+  spinners.forEach(spinner => {
+    spinner.style.transform = 'rotate(' + rotateCount + 'deg)';
+  });
+  requestAnimationFrame(spinSpinner);
+}
+
+export function hideSpinner(spinners) {
+  spinners.forEach(spinner => {
+    spinner.classList.add('display-none');
+  });
+}
```

Comment:
https://github.com/woowacourse/javascript-racingcar/pull/147/files#diff-500d565712f6a3f8e4e4861994f100453f69bd2d265515c75b4591f78375de62R27 에서 `hideSpinner(document.querySelectorAll('.spinner'))`이렇게 호출을 하고 있는데, Controller 입장에서는 View 에게 '스피너를 숨겨줘!'만 해주면 좋을 거 같습니다. 'DOM 을 가져와 스피너를 숨기는 것'까지 View 는 제공을 해주고, 그 실행은 Controller 에서 하면 되고요
```suggestion
// hideSpinners();
export function hideSpinners() {
  const spinners = document.querySelectorAll('.spinner')
  spinners.forEach(spinner => {
    spinner.classList.add('display-none');
  });
}
```